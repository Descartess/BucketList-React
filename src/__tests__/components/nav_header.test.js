import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../components/nav_header';

describe('banner snapshot ', () => {
  const props = {
    handleSelect: jest.fn(),
  };
  it('renders correctly', () => {
    const rendered = renderer.create(
      <Header {...props} />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
