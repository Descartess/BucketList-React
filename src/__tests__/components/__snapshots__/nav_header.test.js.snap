// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`banner snapshot  renders correctly 1`] = `
<nav
  className="navbar navbar-default"
>
  <div
    className="container-fluid"
  >
    <div
      className="navbar-header"
    >
      <a
        className="navbar-brand"
      >
        BucketList
      </a>
    </div>
    <ul
      className="nav navbar-nav navbar-right"
      role={null}
    >
      <li
        className="dropdown"
        style={undefined}
      >
        <a
          aria-expanded={false}
          aria-haspopup={true}
          className="dropdown-toggle"
          href="#"
          id="basic-nav-dropdown"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
        >
          Account Settings
           
          <span
            className="caret"
          />
        </a>
        <ul
          aria-labelledby="basic-nav-dropdown"
          className="dropdown-menu"
          role="menu"
        >
          <li
            className=""
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="menuitem"
              tabIndex="-1"
            >
              Sign Out
            </a>
          </li>
          <li
            className=""
            role="presentation"
            style={undefined}
          >
            <a
              href="#"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="menuitem"
              tabIndex="-1"
            >
              Reset Password 
            </a>
          </li>
        </ul>
      </li>
    </ul>
  </div>
</nav>
`;
